tests/ffi/qm-oom-score-adj/test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+
+. ../common/prepare.sh
+
+export QM_HOST_REGISTRY_DIR="/var/qm/lib/containers/registry"
+export QM_REGISTRY_DIR="/var/lib/containers/registry"
+
+disk_cleanup
+prepare_test
+reload_config
+
+# Copy container image registry to /var/qm/lib/containers
+image_id=$(podman images | grep quay.io/centos-sig-automotive/ffi-tools | awk -F " " '{print $3}')
+if [ ! -d "${QM_HOST_REGISTRY_DIR}" ]; then
+    exec_cmd "mkdir -p ${QM_HOST_REGISTRY_DIR}"
+    exec_cmd "podman push ${image_id} dir:${QM_HOST_REGISTRY_DIR}/tools-ffi:latest"
+fi
+
+podman exec -it qm /bin/bash -c \
+         "podman run -d --replace --name ffi-qm  dir:${QM_REGISTRY_DIR}/tools-ffi:latest \
+          /usr/bin/sleep infinity > /dev/null"
+
+
+QM_PID=$(podman inspect qm --format '{{.State.Pid}}' | tr -d '\r')
```

Comment:
consider using this 
systemctl show -P MainPID qm

Reply:
@Yarboa `systemctl show -P MainPID qm` reports the conmon's pid not the real process' pid, the one that has the oom_score_adj applied